src/components/catalog-filter/catalog-filter.html

```diff
@@ -0,0 +1,22 @@
+<pf-filter-panel config="$ctrl.config">
+  <div class="filter-panel-container">
+    <input type="text" ng-model="$ctrl.keywordFilter.value"
+           class="keyword-filter"
+           placeholder="{{$ctrl.keywordFilter.placeholder}}"
+           ng-keypress="$ctrl.onKeywordKeyPress($event)"
+           ng-disabled="$ctrl.config.totalCount <= 1"/>
+    <div class="category" ng-repeat="filter in $ctrl.filterPanelModel" ng-if="!$first">
```

Comment:
I'm not clear on what this is doing. It's a repeat on filters, but we show the vendor warning for them all? Are we skipping the first because we know it's the keyword filter?